dask/array/utils.py

```diff
@@ -71,12 +75,50 @@ def meta_from_array(x, ndim=None, dtype=None):
     except Exception:
         meta = np.empty((0,) * ndim, dtype=dtype or x.dtype)
 
+    if np.isscalar(meta):
+        meta = np.array(meta)
```

Comment:
Why do we need this block? It breaks 3 CuPy tests: https://github.com/dask/dask/blob/master/dask/array/tests/test_cupy.py#L15-L17

Reply:
I believe that I have resolved this in 83ff1d4

Reply:
Yes, I can confirm. Thanks for that!